apps/roam/src/components/settings/AdminPanel.tsx

```diff
@@ -196,41 +210,87 @@ const AdminPanel = () => {
-            <div className="mx-2 inline-block">
-              <Select
-                items={schemas}
-                onItemSelect={(choice) => {
-                  setShowingSchema(choice);
-                }}
-                itemRenderer={(node, { handleClick, modifiers }) => (
-                  <MenuItem
-                    active={modifiers.active}
-                    key={node.sourceLocalId}
-                    label={node.name}
-                    onClick={handleClick}
-                    text={node.name}
+    <Tabs
+      onChange={(id) => setSelectedTabId(id)}
+      selectedTabId={selectedTabId}
+      renderActiveTabPanelOnly={true}
+    >
+      <Tab
+        id="node-list"
+        title="Node list"
+        panel={
+          <>
+            <p>
+              Context:{" "}
+              <code>
+                {JSON.stringify({ ...context, spacePassword: "****" })}
+              </code>
+            </p>
+            {schemas.length > 0 ? (
+              <>
+                <Label>
+                  Display:
+                  <div className="mx-2 inline-block">
+                    <Select
+                      items={schemas}
+                      onItemSelect={(choice) => {
+                        setShowingSchema(choice);
+                      }}
+                      itemRenderer={(node, { handleClick, modifiers }) => (
+                        <MenuItem
+                          active={modifiers.active}
+                          key={node.sourceLocalId}
+                          label={node.name}
+                          onClick={handleClick}
+                          text={node.name}
+                        />
+                      )}
+                    >
+                      <Button text={showingSchema.name} />
+                    </Select>
+                  </div>
+                </Label>
+                <div>
+                  {loadingNodes ? <Spinner /> : <NodeTable nodes={nodes} />}
+                </div>
+              </>
+            ) : (
+              <p>No node schemas found</p>
+            )}
+          </>
+        }
+      />
+      <Tab
+        id="experiments"
+        title="Experiments"
+        panel={
+          <div className="flex flex-col gap-4 p-1">
+            <Checkbox
+              defaultChecked={
+                extensionAPI.settings.get("use-reified-relations") as boolean
+              }
+              onChange={(e) => {
+                const target = e.target as HTMLInputElement;
+                extensionAPI.settings.set(
+                  "use-reified-relations",
+                  target.checked,
+                );
+              }}
+              labelElement={
+                <>
+                  Reified Relation Triples
+                  <Description
+                    description={
+                      "When ON, relations are read/written as sourceUid:relationBlockUid:destinationUid in [[roam/js/discourse-graph/relations]]."
```

Comment:
I think instead of `sourceUid:relationBlockUid:destinationUid` maybe simpler wording would be better maybe the suggested text in linear?

_“When ON, relations are read/written as reifiedRelationUid in [[roam/js/discourse-graph/relations]].”_

This is what it looks like in ui and requires effort to read it

<img width="1220" height="260" alt="Image" src="https://github.com/user-attachments/assets/f26e97d3-4d92-47f5-b768-490c37d00a46" />